tqdm.py

```diff
@@ -0,0 +1,85 @@
+import time
+
+PROGRESS_BAR_LEN = 20
+
+
+class Tqdm:
+    """
+    Iterable that prints status of other iterable objects
+    print examples:
+        10/20 : 5.03 : 1.99 : ##########.......... : 50%
+        9it : 6.78 : 1.33it/s
+    Attributes:
+        iterable: Iterable object to print his status
+        length: The length extracted from iterable
+    Methods:
```

Comment:
No need for the methods and attributes in the docstring. IPython will take care of that.